polaris-react/src/components/AppProvider/global.css

```diff
@@ -53,18 +53,23 @@
 
 html,
 body {
-  font-size: var(--p-font-size-325);
-  line-height: var(--p-font-line-height-500);
+  font-size: var(--p-font-size-400);
+  line-height: var(--p-font-line-height-600);
   font-weight: var(--p-font-weight-regular);
   font-feature-settings: 'calt' 0;
   letter-spacing: initial;
-  color: var(--p-color-text);
+
+  @media (--p-breakpoints-md-up) {
+    font-size: var(--p-font-size-325);
+    line-height: var(--p-font-line-height-500);
+  }
 }
 
 html,
 body,
 button {
   font-family: var(--p-font-family-sans);
+  color: var(--p-color-text);
```

Comment:
Would the `font-size`/`line-height` updates and added responsiveness/default `button` `color` in the reset be considered a breaking change?